human_analysis/1_process_recount2_data.ipynb

```diff
@@ -48,12 +50,10 @@
     "import pandas as pd\n",
     "import numpy as np\n",
     "import rpy2\n",
```

Comment:
Question: Do we need to import this module explicitly?

Reply:
Tried commenting this `rpy2` module and it doesn't look like you do need this. Thanks for checking